### PR TITLE
New version: B3log.SiYuan version 2.5.0 [FP-D]

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
### Result: Installation Successful
|             | Manifest                 | Add/Remove Programs                   |
| ----------- | ------------------------ | ------------------------------------- |
| Name        | SiYuan | SiYuan 2.5.0    |
| Version     | 2.5.0     | 2.5.0 |
| Publisher   | Yunnan Liandi Technology Co., Ltd.   | Yunnan Liandi Technology Co., Ltd.      |
| ProductCode |           | c1e77cb7-68e1-5adb-ab29-ecf7379a9944    |
#### Auto-updated by [vedantmgoyal2009/winget-pkgs-automation](https://github.com/vedantmgoyal2009/vedantmgoyal2009/tree/main/winget-pkgs-automation) in workflow run [3879](<https://github.com/vedantmgoyal2009/vedantmgoyal2009/actions/runs/3463672311>)

### CODE REVIEW

File: manifests/q/Qingfeng/HeyboxAccelerator/1.1.19/Qingfeng.HeyboxAccelerator.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+PackageIdentifier: Qingfeng.HeyboxAccelerator
+PackageVersion: 1.1.19
+PackageLocale: en-US
+Publisher: Qingfeng
+PublisherUrl: https://open.heybox.hk/
+PublisherSupportUrl: https://acc.xiaoheihe.cn/help/center
+PrivacyUrl: https://api.xiaoheihe.cn/account/acc_privacy_introduce/?type=2
+Author: Qingfeng (Beijing) Technology Co., Ltd.
+PackageName: 小黑盒加速器
```
Comment (stephengillie) [inline manifests/q/Qingfeng/HeyboxAccelerator/1.1.19/Qingfeng.HeyboxAccelerator.locale.en-US.yaml:12]: `PackageName: Heybox Accelerator`

File: manifests/q/Qingfeng/HeyboxAccelerator/1.1.19/Qingfeng.HeyboxAccelerator.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+Tags:
+- accelerate
+- accelerator
+- game
+- gaming
+- network
+- proxy
+# Agreements:
+ReleaseNotes: |-
+  增加实时热搜功能
```
Comment (stephengillie) [inline manifests/q/Qingfeng/HeyboxAccelerator/1.1.19/Qingfeng.HeyboxAccelerator.locale.en-US.yaml:30]: `  Added real-time hot search function`

File: manifests/q/Qingfeng/HeyboxAccelerator/1.1.19/Qingfeng.HeyboxAccelerator.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  网络监控界面优化
```
Comment (stephengillie) [inline manifests/q/Qingfeng/HeyboxAccelerator/1.1.19/Qingfeng.HeyboxAccelerator.locale.en-US.yaml:31]: `   Network monitoring interface optimization`

File: manifests/q/Qingfeng/HeyboxAccelerator/1.1.19/Qingfeng.HeyboxAccelerator.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  增加多平台启动项功能
```
Comment (stephengillie) [inline manifests/q/Qingfeng/HeyboxAccelerator/1.1.19/Qingfeng.HeyboxAccelerator.locale.en-US.yaml:32]: `   Added multi-platform startup item function`

**Comment by stephengillie** [comment]
@vedantmgoyal2009, can the Chinese text in the English localization file be translated into English? I used Google Translate in the above suggestions but feel free to find another translation source.

**Review by stephengillie — CHANGES_REQUESTED** [review]

File: manifests/q/Qingfeng/HeyboxAccelerator/1.1.19/Qingfeng.HeyboxAccelerator.locale.en-US.yaml (hunk 1 above)
Comment (SpecterShell) [reply manifests/q/Qingfeng/HeyboxAccelerator/1.1.19/Qingfeng.HeyboxAccelerator.locale.en-US.yaml:12]: `PackageName` corresponds to the `DisplayName` of the ARP entry the installer wrote to the registry.

**Review by ItzLevvie — DISMISSED** [review]

**Comment by stephengillie** [comment]
Merge conflicts, duplicate of #88280.

**Review by stephengillie — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**